### DISCUSSION TITLE
Is there a way to add events / holidays / reminders to the calendar?

### DISCUSSION THREAD

**Discussion by CathrineWong** [kudrykv/latex-yearly-planner#101] [Q&A]
Hi, I love this calendar.  I'm wondering if there is a way to add customized events / holidays / reminders, such as birthdays, holidays, school/conference meetings, etc.  to the calendar using a cfg file or something similar?

**Comment by kudrykv** [comment]
Hi!
There is no such functionality yet

**Reply by ipapast** [reply]
I've been trying out something, where you can give an ical file and show the events on the daily planner. It works, but I don't have much time to perfect it. For example, if there are too many events (maybe more than 5), not all will show (due to space).
The basic issue is that calendars and timezones are hard. So there are cases missing.

 Currently it looks like this:
<img width="436" alt="image" src="https://github.com/kudrykv/latex-yearly-planner/assets/6989529/2904137e-a8c1-4595-b0da-05952ec89271">

I opened a PR for @kudrykv to check out. https://github.com/kudrykv/latex-yearly-planner/pull/104

If you want to try it out @CathrineWong , you can check my fork [here](https://github.com/ipapast/latex-yearly-planner), and run the command I specify on the PR above:
```
PLANNER_YEAR=2024 \
PASSES=1 \
CFG="cfg/base.yaml,cfg/template_breadcrumb.yaml,cfg/sn_a5x.breadcrumb.default.yaml" \
NAME="sn_a5x.breadcrumb.default" \
ICSFILE="app/components/icsparser/example_calendar.ics" \ 
./single.
```

**Reply by CathrineWong** [reply]
Thank you!  I will give that a try.
